src/app/model/User.ts

```diff
@@ -18,7 +18,7 @@ const defaultUserData: UserInterface = {
   phoneNumber: "",
   email: "",
   location: defaultLocation,
-  organizationUid: 0,
+  organizationUid: "",
```

Comment:
This is the 2nd or 3rd time I've seen someone change this between a string and a number. Have we been able to identify what this should be, and curious if we've checked how this propogates to the rest of the app.

Reply:
When I look at the firebase console, the organizationUid is a string. And the organizations' uid is a string: "1". I think I was getting an error which is why I had to change it, but I can't remember for sure.

Reply:
I can confirm that an error popped up when I changed this back to a number on my local copy.

Reply:
This should be a string, because it is going to refer to a database fk, so that is not ever going to be a number....

Reply:
Cool thanks for clarifying